cpp/ql/src/Likely Bugs/Leap Year/LeapYear.qll

```diff
@@ -69,8 +69,8 @@ abstract class LeapYearFieldAccess extends YearFieldAccess {
    * and it involves an arithmetic operation.
    * In order to avoid false positives, the operation does not includes values that are normal for year normalization.
    *
-   * 1900 - struct tm counts years since 1900
-   * 1980/80 -  FAT32 epoch
+   * 1900 - `struct tm` counts years since 1900
+   * 1980/80 - FAT32 epoch
```

Comment:
The FAT32 epoch doesn't sound like a date that's interesting in most code bases. Perhaps replace or augment with the Unix epoch (1 January 1970).

Reply:
I'll merge this PR now. To be clear, we should base any follow-up changes to the query implementation on actual observations on lgtm.com rather than my opinions about magic numbers in the code.